### PR TITLE
costco.md file created

### PR DESCRIPTION
**Pull request by Nimanita**

### CODE REVIEW

File: companies/costco.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+forbes_rank: 142
+name: Costco Wholesale
+slug: Costco
```
Comment (kelseyhuse30) [inline companies/costco.md:4]:
```suggestion
slug: costco
```

Here's some documentation about slugs - https://support.squarespace.com/hc/en-us/articles/205814578-URL-slugs#toc-url-slug-formatting-tips

They should be all lowercase :)

File: companies/costco.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+forbes_rank: 142
```
Comment (kelseyhuse30) [inline companies/costco.md:2]:
```suggestion
forbes_rank: 14
```

It looks like Costco is #14 .  I need to update this from forbes_rank to fortune_rank 🤦🏻  Sorry about that
https://fortune.com/company/costco/fortune500/

**Review by kelseyhuse30 — CHANGES_REQUESTED** [review]
A few small updates and this is ready to go. Thank you SO MUCH for your research and finding this data for us. Your contribution is greatly appreciated. 🎉

**Comment by Nimanita** [comment]
Yes I will definitely make the change .Should I right forbes rank or fortune rank?

**Comment by kelseyhuse30** [comment]
> Yes I will definitely make the change .Should I write forbes rank or fortune rank?

if you don't mind changing it to `fortune_rank`, that would be fantastic.

**Review by kelseyhuse30 — APPROVED** [review]

**Pull request merged**